### ISSUE TITLE
Manually editing accidentally causing corruption

### ISSUE THREAD

**Issue by ollayf** [ollayf/pe#8]
![Screenshot 2023-04-14 at 5.03.44 PM.png](https://raw.githubusercontent.com/ollayf/pe/main/files/cf93e940-47a1-4762-8165-c58167949b7b.png)

When there are manual inputs and edits that accidentally cause corruption of data

![Screenshot 2023-04-14 at 5.04.39 PM.png](https://raw.githubusercontent.com/ollayf/pe/main/files/7ea510f0-b244-4a86-9203-50a05d03a453.png)

The app does not guide the user as to how to fix this issue. And again similar to the previous issue raised, if they make any more edits, their accidentally corrupted lines of data will be erased. 

This is not a very edge case as in the case where someone forgets to include or exclude one extra ~|~ in their manual editing mode, they may lose alot of their data

**Comment by nus-se-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
This issue mentions that the app does not guide the user as to how to fix this issue.  

But as seen in the User Guide in the image below, there is a guide telling the users the proper format to follow. The guide states that: `~|~` separates the individual component and **It is mandatory to follow the format stated to ensure that the details will be read correctly by the application.**  
Furthermore, the format and examples were given on how to edit the file as well. Therefore, even after reading the user guide and the user stills keys in the wrong format, we decided to print out the affected line and inform the user that the line will be skipped.   

To add on, we also handled the error of wrong formatting in the application gracefully and also informed the user what line in the file was the wrong line. Hence we decided to lower this severity to low as well.

In conclusion, our team decide to reject this bug as this is the expected behavior. Perhaps just a misunderstanding from the user after reading our User Guide.

![image.png](https://raw.githubusercontent.com/nus-cs2113-AY2223S2/pe/master/files/35874b0b-7bb2-4544-b8ab-dc12000745b4.png)

# Items for the Tester to Verify

## :question: Issue response

Team chose [`response.Rejected`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>

## :question: Issue severity

Team chose [`severity.Low`]
Originally [`severity.Medium`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
